templates/base.tmpl

```diff
@@ -1,50 +1,51 @@
-#? stdtmpl | standard
+##? stdtmpl | standard
```

Comment:
Why the double ##?

Reply:
It leads to `templates/base.tmpl(7, 16) Error: invalid indentation`

Reply:
Yes, I saw it. Has a typo come from a paste.